applications/luci-app-ddns/htdocs/luci-static/resources/view/ddns/overview.js

```diff
@@ -0,0 +1,993 @@
+					'click': L.ui.createHandlerFn(this, function() {
+						return callReloadService(section_id).then(L.bind(m.render, m));
+					}),
+					'title': _('Reload this service'),
+				},
+				stop_opt = {
+					'class': 'cbi-button cbi-button-neutral stop',
+					'click': L.ui.createHandlerFn(this, function() {
+
+						return callReloadService(section_id).then(L.bind(m.render, m));
+					}),
+					'title': _('Stop this service'),
+				};
+
+			if (status['_enabled'] == 0 || cfg_enabled == 0)
+				reload_opt['disabled'] = 'disabled';
+
+			if (!resolved[section_id].pid)
+				stop_opt['disabled'] = 'disabled';
+
+			L.dom.content(tdEl.lastChild, [
+				E('button', stop_opt, _('Stop')),
+				E('button', reload_opt, _('Reload')),
+				tdEl.lastChild.childNodes[0],
+				tdEl.lastChild.childNodes[1],
+				tdEl.lastChild.childNodes[2]
+			]);
+
+			return tdEl;
+		};
+
+		o = s.option(form.DummyValue, '_cfg_name', _('Name'));
+		o.modalonly = false;
+		o.textvalue = function(section_id) {
+			return '<b>' + section_id + '</b>';
+		}
+		
+		o = s.option(form.DummyValue, '_cfg_detail_ip', _('Lookup Hostname') + "<br>" + _('Registered IP'));
+		o.rawhtml   = true;
+		o.modalonly = false;
+		o.textvalue = function(section_id) {
+			var host = uci.get('ddns', section_id, 'lookup_host') || _('Configuration Error'),
+				ip = _('No Data');
+			if (resolved[section_id] && resolved[section_id].ip) 
+				ip = resolved[section_id].ip;
+
+			return host + '<br>' + ip;
+		};
+
+		o = s.option(form.Flag, 'enabled', _('Enabled'));
+		o.rmempty  = false;
+		o.editable = true;
+		o.modalonly = false;
+
+		o = s.option(form.DummyValue, '_cfg_update', _('Last Update') + "<br>" + _('Next Update'));
+		o.rawhtml   = true;
+		o.modalonly = false;
+		o.textvalue = function(section_id) {
+			var last_update = _('Never'), next_update = _('Unknown');
+			if (resolved[section_id]) {
+				if (resolved[section_id].last_update)
+					last_update = _(resolved[section_id].last_update);
+				if (resolved[section_id].next_update)
+					next_update =  _(resolved[section_id].next_update);
+			}
+
+			return  last_update + '<br>' + next_update;
+		};
+		
+		s.modaltitle = function(section_id) {
+			return _('DDns Service') + ' » ' + section_id;
+		};
+		
+		o = s.option(form.DummyValue, '_cfg_status', _('Status'));
+		o.modalonly = false;
+		o.textvalue = function(section_id) {
+			var text = '<b>' + _('Not Running') + '</b>';
+
+			if (resolved[section_id] && resolved[section_id].pid)
+				text = '<b>' + _('Running') + '</b> : ' + resolved[section_id].pid;
+
+			return text;
+		}
+
+
+		s.tab('basic', _('Basic Settings'));
+		s.tab('advanced', _('Advanced Settings'));
+		s.tab('timer', _('Timer Settings'));
+		s.tab('logview', _('Log File Viewer'));
+		
+		// TAB: BASIC
+
+		// enabled
+		o = s.taboption('basic', form.Flag, 'enabled', _('Enabled'),_("If this service section is disabled it could not be started." + "<br />" +
+		"Neither from LuCI interface nor from console"));
```

Comment:
Here's another instance of a non-constant expression. Unfortunately our string extraction script is not able to handle concat expressions, so you need to put everything into one string.

Reply:
I think i will put an array to enumerate all of them... This way the string extraction script can detect the different values and still use args in the _() function. Correct? 
Or is better to use switch case?

Reply:
Right, that would work.